### PR TITLE
Round 1.

### PR DESCRIPTION
**Pull request by nirmo-lightricks**

### CODE REVIEW

File: app/src/main/java/com/lightricks/ex1/Common.java [hunk 1]
```diff
@@ -35,13 +35,15 @@ public static boolean hasReadContactsPermission(Context context) {
     public static void glideSetImage(Context context, ImageView ivImage, String imageName) {
         int imageId = getImageId(context, imageName);
         if (imageId == 0){
+            // NOT thread safe! Think about a safe way to preform this operation (the default image calculation)...
```
Comment (razkarl) [inline app/src/main/java/com/lightricks/ex1/Common.java:38]: Good point, though what reason do we have to believe that multiple threads will access this function simultaneously?

File: app/src/main/java/com/lightricks/ex1/Common.java [hunk 2]
```diff
@@ -35,13 +35,15 @@ public static boolean hasReadContactsPermission(Context context) {
...
             // Init defultImageId for the first time
             if (defaultImageId == -1){
                 defaultImageId = getImageId(context, "default_image");
             }
             // Validate defaultImageId
             if (defaultImageId == 0){
                 // Oh no! "default_image.jpeg" is missing!
+                // Use logging here! notify about it.
```
Comment (razkarl) [inline app/src/main/java/com/lightricks/ex1/Common.java:46]: What's the standard logging system we use in Lightricks?

Comment (nirmo-lightricks) [reply app/src/main/java/com/lightricks/ex1/Common.java:46]: We are using Timber, but you can use android default logging utils..

File: app/src/main/java/com/lightricks/ex1/Common.java (hunk 1 above)
Comment (nirmo-lightricks) [reply app/src/main/java/com/lightricks/ex1/Common.java:38]: What reason do we have to believe that multiple threads won't access this function simultaneously?

When writing a "common"/utilities classes which meant to be shared and reusable the public functions should avoid such assumptions. if you have assumptions then document them + verify them if you can.

**Pull request closed without merging**